reactDashboard/src/components/HeadlineBanner/HeadlineBanner.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+const HeadlineBanner = () => {
```

Comment:
The only thing I'd suggest is that if a component doesn't do anything, it would be better to inline the static formatting and bin the component.